### PR TITLE
Add card user terms link

### PR DESCRIPTION
**Pull request by MarySnopok**

### CODE REVIEW

File: text_strings/client/nb.json [hunk 1]
```diff
@@ -2756,6 +2756,7 @@
   "pfs_take_over_third_paragraph_p3": "PLZ_TRANSLATE Betala går fortfarande. Det är inte längre möjligt att fylla på pengar på ditt Gimi-kort.",
   "pfs_take_over_fouth_paragraph_p3": "PLZ_TRANSLATE Ditt Gimi-kort slutar att fungera och din prenumeration kommer fortsätta till det lägre priset.",
   "pfs_take_over_link_paragraph_p3": "PLZ_TRANSLATE Läs hela uppsägningsmeddelandet från Gimi.",
+  "card-user-terms-link": "https://gimi.com/sv/card-issuer-terms/",
```
Comment (gimi-anders) [inline text_strings/client/nb.json:2759]: This should have nb in the url I guess?

File: text_strings/client/nl.json [hunk 2]
```diff
@@ -2756,6 +2756,7 @@
   "pfs_take_over_third_paragraph_p3": "PLZ_TRANSLATE Betala går fortfarande. Det är inte längre möjligt att fylla på pengar på ditt Gimi-kort.",
   "pfs_take_over_fouth_paragraph_p3": "PLZ_TRANSLATE Ditt Gimi-kort slutar att fungera och din prenumeration kommer fortsätta till det lägre priset.",
   "pfs_take_over_link_paragraph_p3": "PLZ_TRANSLATE Läs hela uppsägningsmeddelandet från Gimi.",
+  "card-user-terms-link": "https://gimi.com/sv/card-issuer-terms/",
```
Comment (gimi-anders) [inline text_strings/client/nl.json:2759]: nl in the url?

File: text_strings/client/en.json [hunk 3]
```diff
@@ -2756,6 +2756,7 @@
   "pfs_take_over_third_paragraph_p3": "PLZ_TRANSLATE Betala går fortfarande. Det är inte längre möjligt att fylla på pengar på ditt Gimi-kort.",
   "pfs_take_over_fouth_paragraph_p3": "PLZ_TRANSLATE Ditt Gimi-kort slutar att fungera och din prenumeration kommer fortsätta till det lägre priset.",
   "pfs_take_over_link_paragraph_p3": "PLZ_TRANSLATE Läs hela uppsägningsmeddelandet från Gimi.",
+  "card-user-terms-link": "https://gimi.com/sv/card-issuer-terms/",
```
Comment (gimi-anders) [inline text_strings/client/en.json:2759]: en in the url?

**Review by gimi-anders — CHANGES_REQUESTED** [review]

File: text_strings/client/en.json (hunk 3 above)
Comment (MarySnopok) [reply text_strings/client/en.json:2759]: @gimi-anders  it doesnt look like there are localised options of this page on our site , I can change it but text is in english always. I will apply the change

Comment (MarySnopok) [reply text_strings/client/en.json]: Changed 🎉

Comment (gimi-anders) [reply text_strings/client/en.json]: Ok, should we ask product about that? If it is supposed to be like that?

Comment (MarySnopok) [reply text_strings/client/en.json]: @gimi-anders  asked

**Comment by MarySnopok** [comment]
@gimi-anders  Got a reply it is a single link for all locals : [https://gimitheapp.slack.com/archives/C05CN0ECQ9X/p1692795683283569](https://gimitheapp.slack.com/archives/C05CN0ECQ9X/p1692795683283569)

**Review by gimi-anders — APPROVED** [review]

**Pull request merged**